

processing file: ClusterFinder.module.Rmd
  |                                                                              |                                                                      |   0%  |                                                                              |....                                                                  |   6%
   inline R code fragments

  |                                                                              |........                                                              |  11%
label: setup (with options) 
List of 1
 $ include: logi FALSE

  |                                                                              |............                                                          |  17%
  ordinary text without R code

  |                                                                              |................                                                      |  22%
label: init (with options) 
List of 3
 $ eval  : symbol F
 $ echo  : symbol T
 $ engine: chr "bash"

  |                                                                              |...................                                                   |  28%
  ordinary text without R code

  |                                                                              |.......................                                               |  33%
label: populate_meta_data_database (with options) 
List of 3
 $ eval   : logi TRUE
 $ echo   : symbol F
 $ results: symbol F

── Attaching packages ─────────────────────────────────────── tidyverse 1.3.0 ──
✔ ggplot2 3.3.3     ✔ purrr   0.3.4
✔ tibble  3.1.0     ✔ dplyr   1.0.5
✔ tidyr   1.1.3     ✔ stringr 1.4.0
✔ readr   1.4.0     ✔ forcats 0.5.1
── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()
Quitting from lines 48-112 (ClusterFinder.module.Rmd) 
Error in file(filename, "r", encoding = encoding) : 
  cannot open the connection
Calls: <Anonymous> ... withCallingHandlers -> withVisible -> eval -> eval -> source -> file
In addition: Warning message:
In file(filename, "r", encoding = encoding) :
  cannot open file 'assets/scTools.r': No such file or directory

Execution halted
